shared-src/docs-developer-guide.rst

```diff
@@ -340,3 +340,87 @@ To exclude an built-in proselint check,
 specify the check name in the check list
 in the function :py:func:`exclude_checks`
 in the file :file:`style-test.py`.
+
+.. _docs-pdf:
```

Comment:
This feels like it'd work better in docs-tech-guide right below Style-test your work? What do you think?

Reply:
Yes, agreed this should be a part of workflow. Made the changes!